### PR TITLE
Schnusch unstable

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

File: nix_update/eval.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def eval_expression(import_path: str, attr: str) -> str:
       line = position.line;
       urls = pkg.src.urls or null;
       url = pkg.src.url or null;
-      rev = pkg.src.url.rev or null;
+      rev = pkg.src.rev or null;
```
Comment (zowoq) [inline nix_update/eval.py:68]: Cherry picked this commit to https://github.com/Mic92/nix-update/pull/97

**Comment by figsoda** [comment]
What is the status of this pr? should we contribute to this instead of #76?

File: nix_update/eval.py [hunk 2]
```diff
@@ -59,13 +59,13 @@ def eval_expression(import_path: str, attr: str) -> str:
         builtins.unsafeGetAttrPos "src" pkg;
     in {{
       name = pkg.name;
-      old_version = (builtins.parseDrvName pkg.name).version;
+      old_version = pkg.version or (builtins.parseDrvName pkg.name).version;
```
Comment (figsoda) [inline nix_update/eval.py:62]:
```suggestion
      old_version = lib.getVersion pkg;
```

**Comment by Mic92** [comment]
> What is the status of this pr? should we contribute to this instead of #76?

It was a rebased version of that, but it also got stalled. Closing this for now, in case someone wants to pick it up, be my guest.

**Pull request closed without merging**